src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -73,6 +73,10 @@ private sealed class ConsoleLogsSubscription
     [Parameter]
     public string? ResourceName { get; set; }
 
+    [Parameter]
+    [SupplyParameterFromQuery(Name = "hideLogDate")]
+    public bool? HideLogDate { get; set; }
```

Comment:
It could be, but I like leaving it nullable because the query string is missing when false. The explicit conversion from `bool?` to `bool` later in the code makes it obvious what behavior is when the query string isn't present.